### ISSUE TITLE
Create CODE_OF_CONDUCT.md file

### ISSUE THREAD

**Issue by payaldutta** [Learn-Write-Repeat/Flutter#1]
I want to create a code of conduct file for those who are new to open source. @abhishekpatelmc  Can I work on this?

**Comment by abhishekpatelmc** [comment]
Yes, you can update it. I will merge the pull request according to the need 
if the files seem helpful I will merge it
